app.js

```diff
@@ -1,13 +1,18 @@
 const express = require('express');
-const itemsRouter = require('./routes/items');
-const usersRouter = require('./routes/users');
+const middleware = require('./modules/middleware.module');
 
 const app = express();
 
-app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
+// middleware functions
+app.use(middleware.helmet);
+app.use(middleware.ratelimit);
+app.use(express.json()); // @todo store also in middleware module
+app.use(express.urlencoded({ extended: false })); // @todo store also in middleware module
 
-app.use('/items', itemsRouter);
-app.use('/users', usersRouter);
+// routes
+app.use('/items', require('./routes/items'));
+app.use('/users', require('./routes/users'));
 
 module.exports = app;
+
+// todo: swagger, error handling, logging, concurrency (performance), editorconfig (brauchen wir gar nicht oder?!), cronjobs module?
```

Comment:
die ToDos wuerde ich in die README setzen, was wir noch verbauen wollen (editorconfig kann gerne rein)
Bitte kein Deutsch im code verwenden :)